### ISSUE TITLE
With the 3.0.0-beta.x code, can the text attribute be made dynamic, so changes to it are reflected in the tooltip?

### ISSUE THREAD

**Issue by BillyRayPreachersSon** [sir-dunxalot/ember-tooltips#225]
If the text attribute of a tooltip is passed from a property, the tooltip will correctly show the value of that property.

However, if the property value is updated before the tooltip is shown, the old value will show.

Would it be possible to change this behaviour?

My use case: I want to show a tooltip on a success or error scenario, with different text for each. With the current implementation, I've got to use two tooltips for this, with the text hard-coded. If I could set the text before showing the tooltip, I'd only need to have a single tooltip in the code.

**Comment by alvincrespo** [comment]
@sir-dunxalot Any way I can help with making this happen?

It looks like we only create the tooltip once:

https://github.com/sir-dunxalot/ember-tooltips/blob/3.0.0-beta.2/addon/components/ember-tooltip-base.js#L303

https://github.com/sir-dunxalot/ember-tooltips/blob/3.0.0-beta.2/addon/components/ember-tooltip-base.js#L164

Upon `didInsertElement`. Looks like we should destroy the tooltip and recreate it if the text changes.

Thoughts on how you would move forward with this?

**Comment by alvincrespo** [comment]
Cloned the repo and checked out the 3.0.0-beta.2 tag and wasn't able to get the dummy app working:

![](https://ac-screenshots.s3.amazonaws.com/not-working.gif)

My environment:

```
➜  ember-tooltips git:(2488560) ✗ ember -v
WARNING: Node v9.2.0 is not tested against Ember CLI on your platform. We recommend that you use the most-recent "Active LTS" version of Node.js.
ember-cli: 2.14.2
node: 9.2.0
os: darwin x64
```

**Comment by alvincrespo** [comment]
I also changed my env to 7.0.0 and got the same after reinstalling assets:

```
➜  ember-tooltips git:(2488560) ember -v
ember-cli: 2.14.2
node: 7.0.0
os: darwin x64
```

**Comment by maxfierke** [comment]
Fixed in `3.0.0-beta.4`
